### PR TITLE
Sequential Reads and Set/Unset status Register

### PR DESCRIPTION
**Pull request by mitch-fuel**
As part of #395, this PR updates the SRW and SRWQ to allow for sequential reads for quad words and populates a provided register with whether the word was set/unset in state.

### CODE REVIEW

**Review by Voxelot — APPROVED** [review]
LGTM! cc @adlerjohn 

note: we'll likely want to update our interpreter storage trait in the VM to support fetching a range of slots.

**Comment by Voxelot** [comment]
@mitch-fuel I should've mentioned this earlier, but in the future, we should wait for either @adlerjohn or multiple approvers before merging spec changes.

File: specs/vm/instruction_set.md
Comment (adlerjohn) [inline specs/vm/instruction_set.md:1425]: Should this be `MEM[$rC, 32]`?

File: specs/vm/instruction_set.md
Comment (adlerjohn) [inline specs/vm/instruction_set.md:1426]: Format table with markdown table formatter

File: specs/vm/instruction_set.md
Comment (adlerjohn) [inline specs/vm/instruction_set.md:1438]: I would use `true` and `false` here since those are well-defined.

File: specs/vm/instruction_set.md
Comment (adlerjohn) [inline specs/vm/instruction_set.md:1445]: Should this be `MEM[$rC, 32 * $rD]`?

File: specs/vm/instruction_set.md
Comment (mitch-fuel) [reply specs/vm/instruction_set.md:1425]: yes, it should be.

**Pull request merged**
